docs/source/tokenizer_summary.rst

```diff
@@ -1,223 +1,172 @@
-Tokenizer summary
+Summary of the tokenizers
```

Comment:
this is more in line with "Summary of the models" and "Summary of the tasks" in my opinion.